### ISSUE TITLE
Add theme options to preferences

### ISSUE THREAD

**Issue by cbrnr** [cbrnr/mnelab#344]
I think it would be nice to have an option to select the theme in the preferences dialog. It should include "light", "dark", and "auto". The caveat at the moment is that theming only really works for macOS (dynamically) and KDE (before starting the app) and maybe GNOME, but on Windows it's always the light theme.

**Comment by cbrnr** [comment]
For manual theme switching, I think we need to manually define the dark palette like e.g. [here](https://github.com/DavidoTek/ProtonUp-Qt/blob/main/pupgui2/constants.py#L52-L68) or [here](https://github.com/qt/qtbase/blob/dev/src/gui/kernel/qplatformtheme.cpp#L355).
